### PR TITLE
Project tabs

### PR DESCRIPTION
**Pull request by nirparai**
Edit Project

- Accordion to tabs
- Project Details UI updates according to Figma

### CODE REVIEW

File: src/containers/my-projects/EditProjectPage.js [hunk 1]
```diff
@@ -41,57 +41,63 @@ export default function EditProjectPage() {
...
+                      <ProjectInfoEdit project={user.currentProject} />
+                    </Tab.Pane>
+                    <Tab.Pane eventKey="impacts">
+                      <ImpactsEdit project={user.currentProject} />
+                    </Tab.Pane>
+                    <Tab.Pane eventKey="outcomes">
+                      <OutcomesEdit project={user.currentProject} />
+                    </Tab.Pane>
+                    <Tab.Pane eventKey="beneficiary-groups">
+                      <BeneficiaryGroupsEdit project={user.currentProject} />
```
Comment (GiffenGood1) [inline src/containers/my-projects/EditProjectPage.js:95]: <BeneficiaryGroupsEdit
                        project={user.currentProject}
                        beneficiaries={user.currentProject.beneficiaries}
                        orgId={user.currentProject.org_id}
                      />
This line needs to be changed to this ^

Comment (nirparai) [reply src/containers/my-projects/EditProjectPage.js:95]: Great, will update

**Comment by GiffenGood1** [comment]
Great work, just a few changes so it looks a bit more consistent.
-remove the fieldset border around each of the forms (also the legend for the beneficary groups form)
-make the save buttons consistent on each form (make them all similar to the figma and not so wide)
- try to make it so the modal doesn't change height so much when you changes tabs. Maybe with a min-height but I'm not too sure on best way to do this ...

**Comment by nirparai** [comment]
the figma prototype has next button. 
Do you wan it removed in all tabs ?

**Comment by GiffenGood1** [comment]
yes please

**Pull request merged**